posts/2017-03-15-faunadb-serverless-authentication.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Using Serverless Authentication Boilerplate with FaunaDB
+description: FaunaDB is a security aware database, so we pass user-specific access tokens to it via a custom authorizer.
```

Comment:
Not entirely sure about the description since this is SEO related and might be seen shown in the Google result.

Maybe this `description` can be refined a little bit...